CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+* @armfazh @cjpatton @claucece @Lekensteyn @wbl
```

Comment:
Should we remove `wbl` while at it?

Reply:
I'd say let's do it in a later PR.

Reply:
On second thought, now is good a time as any. Go for it.